python/ray/rllib/env/base_env.py

```diff
@@ -78,26 +78,20 @@ def to_base_env(env,
                     make_env=None,
                     num_envs=1,
                     remote_envs=False,
-                    async_remote_envs=False):
+                    remote_worker_env_timeout_ms=0):
         """Wraps any env type as needed to expose the async interface."""
 
         from ray.rllib.env.remote_vector_env import RemoteVectorEnv
-        if (remote_envs or async_remote_envs) and num_envs == 1:
+        if remote_envs and num_envs == 1:
             raise ValueError(
                 "Remote envs only make sense to use if num_envs > 1 "
                 "(i.e. vectorization is enabled).")
-        if remote_envs and async_remote_envs:
-            raise ValueError("You can only specify one of remote_envs or "
-                             "async_remote_envs.")
 
         if not isinstance(env, BaseEnv):
             if isinstance(env, MultiAgentEnv):
                 if remote_envs:
                     env = RemoteVectorEnv(
-                        make_env, num_envs, multiagent=True, sync=True)
-                elif async_remote_envs:
-                    env = RemoteVectorEnv(
-                        make_env, num_envs, multiagent=True, sync=False)
+                        make_env, num_envs, multiagent=True, remote_worker_env_timeout_ms=remote_worker_env_timeout_ms)
```

Comment:
Consider `inference_batch_timeout_ms`

Reply:
Not sure that name catches the point better? In my mind, inference is not tied with env reset/step but with getting the next action from the model, and remote is needed in the name so that we have a hint it is tied to remote envs.

Reply:
Hm, how about remote_worker_env_batch_wait_ms? At first glance, I think the current name does not necessarily cause one to think this is for batching, but rather for some error-related timeout.

Reply:
Done